### PR TITLE
Translation infrastructure; propagation through the views. Also fix E…

### PR DESCRIPTION
**Pull request by EndymionJkb**
…ditCap (show correct value, allow 0, no decimal points)

Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/components/Modal/CreatePool.vue [hunk 1]
```diff
@@ -62,18 +62,19 @@
...
   data() {
     return {
       stepTitles: [
-        'Select Tokens',
-        'Set Weights',
-        'Deposit',
-        'Set Swap Fee',
-        'Preview'
+        i18n.tc('selectTokens'),
+        i18n.tc('set=Weights'),
```
Comment (bonustrack) [inline src/components/Modal/CreatePool.vue:74]: Some typo here
```suggestion
        i18n.tc('set=Weights'),
```

File: src/components/Tabs.vue [hunk 2]
```diff
@@ -3,41 +3,43 @@
...
+import i18n from '@/i18n';
+
 export default {
   props: ['pool'],
   computed: {
     items() {
       const items = [
         {
-          name: 'Balances',
+          name: i18n.tc('balances'),
```
Comment (bonustrack) [inline src/components/Tabs.vue:14]: You could do it like this:
```suggestion
          name: this.$t('balances'),
```
This is more consistent with the way you do it on the view template (`$t('key')`) it also avoid importing `@/i18n`

File: src/locales/en-US.json [hunk 3]
```diff
@@ -39,40 +73,111 @@
...
+    "vulnerableMid": "s are not correct, you are vulnerable to loss of funds through arbitrage. You can click on",
+    "vulnerablePost": "to toggle manual input",
+    "waitingConfirmation": "Waiting for confirmation",
     "walletBalance": "Wallet balance",
     "weight": "Weight",
-    "weightConstraints": "Weights should be numbers from 2 to 98. Total weight should not exceed 100.",
+    "weightConstraints": "Denormalized weights should be numbers from 2 to 98. Total weight should not exceed 100.",
     "weights": "Weights",
-    "weightsPct": "Weights Percent"
+    "weightsPct": "Weights Percent",
```
Comment (bonustrack) [inline src/locales/en-US.json:179]: Will be better to use Sentence case here instead of Title Case
```suggestion
    "weightsPct": "Weights percent",
```

Comment (bonustrack) [reply src/locales/en-US.json:179]: Same here https://github.com/balancer-labs/pool-management-vue/pull/160/files#diff-65fd212bf11fb0785dba0e4951aca7edR13

**Review by bonustrack — COMMENTED** [review]
Great work! I've added some suggestions.

File: src/components/Tabs.vue (hunk 2 above)
Comment (EndymionJkb) [reply src/components/Tabs.vue]: Thanks! I struggled with that - finally figured it out, but didn't think to try this.$t

**Review by bonustrack — APPROVED** [review]
Looking great!

File: src/components/Chart.vue [hunk 4]
```diff
@@ -29,15 +29,15 @@ import * as TV from 'lightweight-charts';
 
 const items = [
   {
-    name: 'Liquidity',
+    name: this.$t('liquidity'),
```
Comment (bonustrack) [inline src/components/Chart.vue:32]: There is some issue here, "this" is not available at this level, you will need to have this stored inside the `export default {`

Comment (EndymionJkb) [reply src/components/Chart.vue:32]: Is this where it might be better to use my original i18n.tc solution?

**Pull request merged**